### PR TITLE
31 pairing wallet

### PR DESCRIPTION
**Pull request by MaikelW**

### CODE REVIEW

**Review by Henjer — APPROVED** [review]
ok!

File: BluckurWallet/BluckurWallet/UILayer/HomePage.xaml [hunk 1]
```diff
@@ -29,12 +30,7 @@
         </ContentPage.Content>
     </ContentPage>
 
+    <local:HelpPage Title="Help" Icon="ic_help.png"/>
```
Comment (Errors4l) [inline BluckurWallet/BluckurWallet/UILayer/HomePage.xaml]: I like this change.

**Review by Errors4l — APPROVED** [review]
Looks good.

File: BluckurWallet/BluckurWallet/UILayer/HomePage.xaml (hunk 1 above)
Comment (MaikelW) [reply BluckurWallet/BluckurWallet/UILayer/HomePage.xaml]: Velly Gut

**Pull request merged**